hazelcast-openshift-centos/Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM centos:7
+MAINTAINER Hazelcast, Inc. Integration Team <info@hazelcast.com>
+
+ENV HZ_HOME /opt/hazelcast/
+ENV HZ_CP_MOUNT ${HZ_HOME}/external
+ENV JAVA_HOME /usr/lib/jvm/java
+ENV LANG en_US.utf8
+
+ENV USER_NAME=hazelcast
+ENV USER_UID=10001
+
+ENV HZ_VERSION 3.8.6
+
+ARG HZ_KUBE_VERSION=1.0.0
+ARG HZ_MAVEN_DIR=${HZ_VERSION}
+ARG REPOSITORY_URL=https://repository-hazelcast-l337.forge.cloudbees.com
+
+LABEL name="hazelcast/hazelcast-openshift-centos" \
+      vendor="Hazelcast, Inc." \
+      version="7" \
+      architecture="x86_64" \
+      release="${HZ_VERSION}" \
+      url="http://www.hazelcast.com" \
+      summary="Hazelcast Openshift Image, based on CENTOS 7" \
+      description="Starts a standalone Hazelcast server instance to form a cluster based on kubernetes discovery inside Openshift" \
+      io.k8s.description="Starts a standalone Hazelcast server instance to form a cluster based on kubernetes discovery inside Openshift" \
+      io.k8s.display-name="Hazelcast" \
```

Comment:
It is better to mention `Hazelcast Enterprise` instead of `Hazelcast` in where needed. We actually uses `Hazelcast Enterprise` in that image.